src/mon/PGMapDigest.cc

```diff
@@ -0,0 +1,52 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*- 
```

Comment:
Wondering if this change is related to the PR.

Reply:
ha, nope, no idea where that came from